src/components/FileList/treeTable.js

```diff
@@ -17,13 +17,6 @@ import PivotCustomComponent from './PivotCustomComponent'
  */
 export default Component => {
   const wrapper = class RTTreeTable extends React.Component {
-    constructor(props) {
```

Comment:
I guess that this updates are due the eslint configuration, right?

Reply:
Not exactly. I saw that we're using arrow functions so the bind of 3 is not necessary, making the entire constructor useless

Reply:
Ok, so this shouldn't be part of this specific issue i guess, but actually there's any issue that address this.
Maybe in #287 would be more appropriated?